File: JavaScript/exercicioManzandoPg46C.js
function somaPares(){
    let contador = 2
    let soma = 0
    while(contador <= 500){
        soma += contador
        contador +=2
    }

    // while(contador <= 500){
    //     if(contador%2==0){
    //         soma += contador
    //     }
    //     contador++
    // }

    document.getElementById("mostra").innerHTML = `A soma dos número pares de 1 ate 500 é ${soma}`
}

File: Java/manzando/exercicioManzandoPg46C.java
package manzando;

public class exercicioManzandoPg46C {
	public static void main(String[] args) {
		int contador = 2, soma = 0;
			    while(contador <= 500){
			        soma += contador;
			        contador +=2;
			    }
		System.out.println("A soma dos número pares de 1 ate 500 é "+soma);
	}
}
